sqrl-tools/sqrl-cli/src/main/java/com/datasqrl/cmd/PackageBootstrap.java

```diff
@@ -192,6 +185,15 @@ public SqrlConfig bootstrap(Repository repository, ErrorCollector errors,
     return postProcess.apply(sqrlConfig);
   }
 
+  // Currently the compilation fails with a nice stack trace when there are multiple package.jsons
+  // Normally we want to merge these jsons.
+  // This method is a temporary solution to give a meaningful error message to the user until we fix this behavior.
+  private void detectDuplicateConfigs(ErrorCollector errors, List<Path> configFiles) {
+    if (configFiles.stream().anyMatch(path -> path.equals(rootDir.resolve(DEFAULT_PACKAGE)))) {
+      errors.fatal("Both the project and the profile have a package.json defined.");
```

Comment:
This check can be dropped for a mutable configFiles. Being able to override a profile's package json will be important for flexibility and we should attempt to preserve that functionality.

Reply:
So, originally, I wanted to include this check to provide better clarity since this scenario is already failing. However, upon reconsideration, I realize it might actually create more issues than it resolves. Therefore, I've decided to remove it.